apps/hash-api/src/integrations/linear/oauth.ts

```diff
@@ -237,29 +253,58 @@ export const oAuthLinearCallback: RequestHandler<
         entityTypeIds: [systemEntityTypes.linearIntegration.entityTypeId],
         webId: userAccountId as WebId,
         properties: linearIntegrationProperties,
-        relationships: createDefaultAuthorizationRelationships(authentication),
+        relationships: [
+          ...createDefaultAuthorizationRelationships(authentication),
+          {
+            relation: "editor",
+            subject: {
+              kind: "account",
+              subjectId: linearBotAccountId,
+            },
+          },
+        ],
       },
     );
 
+    const entityUuid = extractEntityUuidFromEntityId(
+      linearIntegrationEntity.entityId,
+    );
+
+    await createPolicy(req.context.graphApi, authentication, {
+      name: `linear-integration-bot-view-entity-${entityUuid}`,
+      principal: {
+        type: "actor",
+        actorType: "machine",
+        id: linearBotAccountId,
+      },
+      effect: "permit",
+      actions: ["viewEntity"],
+      resource: {
+        type: "entity",
+        id: entityUuid,
+      },
+    });
+
+    await createPolicy(req.context.graphApi, authentication, {
+      name: `linear-integration-user-view-entity-${entityUuid}`,
+      principal: {
+        type: "actor",
+        actorType: "user",
+        id: userAccountId as UserId,
```

Comment:
Instead of casting it here to `UserId` we could do that in line 204 already
```diff
  const userAccountId = extractEntityUuidFromEntityId(
    actorEntityId,
-  ) as ActorEntityUuid;
+  ) as UserId;
```

Reply:
Done in https://github.com/hashintel/hash/pull/7407/commits/89c8f1c92d2d684551f93b9a9159c5edc3ef148b